recipes/nextflow/build.sh

```diff
@@ -1,3 +1,4 @@
 mkdir -p $PREFIX/bin
 chmod 755 nextflow
 cp nextflow $PREFIX/bin
+$PREFIX/bin/nextflow -download
```

Comment:
The installed "nextflow" binary is a mere wrapper script that launches a Java application. The `-download` option pulls the required JAR dependencies in the home directory, otherwise they will downloaded the very first time the user run the `nextflow` command. This is needed to pull the application dependencies so that they are included in the nextflow biocontainer created by this recipe (otherwise they will downloaded each time the containers is run making it useless).